Prevent Deprecation Warning to users

## User get DeprecationWarnings

- Operating system: Windows 10

Run install and run [nox](https://nox.thea.codes/en/stable/) on attached archive
[example_python.zip](https://github.com/neo4j/neo4j-python-driver/files/5842343/example_python.zip)

- Expected behavior
pylint should analyze the code and report user erros

- Actual behavior
pylint reports issue in neo4j driver as well!

```
================================================= warnings summary =================================================
<unknown>:339
  <unknown>:339: DeprecationWarning: invalid escape sequence \*

<unknown>:381
  <unknown>:381: DeprecationWarning: invalid escape sequence \*

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============================================= short test summary info ==============================================
FAILED example.py::PYLINT
```

--- a/neo4j/work/simple.py
+++ b/neo4j/work/simple.py
@@ -370,7 +370,7 @@ class Session(Workspace):
             with driver.session() as session:
                 values = session.read_transaction(get_two_tx)
 
-        :param transaction_function: a function that takes a transaction as an argument and does work with the transaction. `tx_function(tx, \*args, \*\*kwargs)`
+        :param transaction_function: a function that takes a transaction as an argument and does work with the transaction. `tx_function(tx, *args, **kwargs)`
         :param args: arguments for the `transaction_function`
         :param kwargs: key word arguments for the `transaction_function`
         :return: a result as returned by the given unit of work
@@ -395,7 +395,7 @@ class Session(Workspace):
                 node_id = session.write_transaction(create_node_tx, "example")
 
 
-        :param transaction_function: a function that takes a transaction as an argument and does work with the transaction. `tx_function(tx, \*args, \*\*kwargs)`
+        :param transaction_function: a function that takes a transaction as an argument and does work with the transaction. `tx_function(tx, *args, **kwargs)`
         :param args: key word arguments for the `transaction_function`
         :param kwargs: key word arguments for the `transaction_function`
         :return: a result as returned by the given unit of work

